keyboards/handwired/clean60/matrix.c

```diff
@@ -0,0 +1,514 @@
+/*
```

Comment:
This look like it is very much based on the ergodox's matrix.

You may want to check out the newer code for it, as it's quite updated compared to what you're using.